ckanext/datitrentinoit/profile.py

```diff
@@ -0,0 +1,83 @@
+  
+import logging
+
+from rdflib.namespace import Namespace, RDF, SKOS
+from rdflib import URIRef, BNode, Literal
+
+import ckan.logic as logic
+
+from ckanext.dcat.profiles import RDFProfile, DCAT, LOCN, VCARD, DCT, FOAF, ADMS, OWL, SCHEMA, TIME
+from ckanext.dcatapit.dcat.profiles import remove_unused_object
+
+DCATAPIT = Namespace('http://dati.gov.it/onto/dcatapit#')
+
+log = logging.getLogger(__name__)
+
+
+class DatitrentinoitProfile(RDFProfile):
+    """
+    Customization of DCATAP-IT for Regione Umbria
+    """
+
+    def parse_dataset(self, dataset_dict, dataset_ref):
+        return dataset_dict
+
+    def graph_from_dataset(self, dataset_dict, dataset_ref):
+        self._replace_poc_with_publisher(dataset_dict, dataset_ref)
+        self._augment_publisher_info(dataset_dict, dataset_ref)
```

Comment:
`graph_from_dataset()` is the needed extension point. The source code you are copying from is doing a couple of things, which are in the name of methods called.
I guess we only need a single method there that will replace the poc only (maybe a `replace_poc` may be a name informative enough). The current name is misleading.